torch/csrc/jit/runtime/register_prim_ops.cpp

```diff
@@ -2727,7 +2727,7 @@ RegisterOperators reg2({
     DEFINE_UNARY_INT_OP(aten::factorial, factorial(a), int),
     DEFINE_UNARY_FLOAT_OP(aten::isnan, std::isnan(a), bool),
     DEFINE_UNARY_FLOAT_OP(aten::isfinite, std::isfinite(a), bool),
-    DEFINE_UNARY_FLOAT_OP(aten::isinf, std::isinf(a), bool),
+    DEFINE_UNARY_FLOAT_OP(aten::isinf, bool(std::isinf(a)), bool),
```

Comment:
accidental revert ?

Reply:
I'm not sure why (maybe it was my compiler toolchain/libc++, I just switched it), but `isinf` was returning an `int` instead of a `bool` for me. Casting it here should be save either way.